microsoft/testsuites/osu/osusuite.py

```diff
@@ -0,0 +1,130 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import re
+import os
+from pathlib import Path
+
+from assertpy import assert_that
+
+from lisa import (
+    LisaException,
+    Logger,
+    Node,
+    SkippedException,
+    TestCaseMetadata,
+    TestSuite,
+    TestSuiteMetadata,
+    constants,
+    simple_requirement,
+    CustomScript,
+    CustomScriptBuilder
+)
+from lisa.features import Gpu, GpuEnabled, SerialConsole
+from lisa.features.gpu import ComputeSDK
+from lisa.operating_system import Debian
+from lisa.tools import Lspci, NvidiaSmi, Pip, Python, Reboot, Service, Tar, Wget
+from lisa.util import get_matched_str
+
+_osu_mpi_location = ("https://mvapich.cse.ohio-state.edu/download/mvapich/mv2/mvapich2-2.3.7-1.tar.gz")
+_osu_mpi_file_name = "mvapich2.tgz"
+
+_osu_bench_location = ("http://mvapich.cse.ohio-state.edu/download/mvapich/osu-micro-benchmarks-6.2.tar.gz")
```

Comment:
Global variables are capital snake case. More guidelines can be found here: https://mslisa.readthedocs.io/en/main/write_test/guidelines.html

Reply:
Understandable, but I am just using the format in this test: https://github.com/microsoft/lisa/blob/ee11364108da6c6e68cef988d11c9301d41e55ed/microsoft/testsuites/gpu/gpusuite.py